### PR TITLE
tests_l2: Update gpu readme for hwinfo

### PR DESCRIPTION
**Pull request by vbedida79**
Added hwinfo readme details
based on https://github.com/intel/intel-technology-enabling-for-openshift/pull/155
Signed-off-by: vbedida79 <veenadhari.bedida@intel.com>

### CODE REVIEW

File: tests/l2/README.md [hunk 1]
```diff
@@ -67,6 +68,38 @@ $ oc logs intel-dgpu-clinfo-56mh2
...
+  [Created at pci.386]
+  Unique ID: ug9x.K5dXxUXtHX2
+  Parent ID: QGIh.mr2N3fBJq5F
+  SysFS ID: /devices/pci0000:30/0000:30:02.0/0000:31:00.0/0000:32:00.0/0000:33:00.0/0000:34:08.0/0000:35:00.0/0000:36:01.0/0000:37:00.0
+  SysFS BusID: 0000:37:00.0
+  Hardware Class: graphics card
+  Model: "Intel VGA compatible controller"
+  Vendor: pci 0x8086 "Intel Corporation"
+  Device: pci 0x56c1
+```
```
Comment (uMartinXu) [inline tests/l2/README.md:102]: What we need is the i915 driver information shown by hwinfo like
  
Driver: "i915"
  Driver Modules: "i915"
  Memory Range: 0x23fe7e000000-0x23fe7fffffff (ro,non-prefetchable)
  Memory Range: 0x236000000000-0x237fffffffff (ro,non-prefetchable)
  IRQ: 138 (447 events)
  Module Alias: "pci:v00008086d00000BD5sv00008086sd00000000bc03sc80i00"
  Driver Info #0:
    Driver Status: i915 is active
    Driver Activation Cmd: "modprobe i915"
  Config Status: cfg=new, avail=yes, need=no, active=unknown
  Attached to: #26 (PCI bridge)

Should we provision the driver and then run the hwinfo?

Comment (vbedida79) [reply tests/l2/README.md:102]: yes we should provision driver. this can be good l1 test case.

Comment (vbedida79) [reply tests/l2/README.md:113]: updated and added complete output

**Review by hershpa — APPROVED** [review]

**Comment by uMartinXu** [comment]
It looks good to me. Thanks!

**Pull request merged**